### PR TITLE
Add new provinces

### PR DESCRIPTION
**Pull request by yujinyuz**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -631,6 +631,33 @@ Replace `platerecognizer/alpr` by `platerecognizer/alpr:thailand` in the [instal
...
+| Prince Edward Islands     | ca-pe       |
+| Saskatchewan              | ca-sk       |
+| Alberta                   | ca-ab       |
+| Newfoundland and Labrador | ca-nl       |
+
+### United Arab Emirates
+
+| State                     | Region Code |
+| ------------------------- | ----------- |
+| Abu Dhabi                 | ca-on       |
```
Comment (marcbelmont) [inline source/index.html.md]: `ca-on` that can't be right.

File: source/index.html.md [hunk 2]
```diff
@@ -631,6 +631,33 @@ Replace `platerecognizer/alpr` by `platerecognizer/alpr:thailand` in the [instal
...
+| Ontario                   | ca-on       |
+| Quebec                    | ca-qc       |
+| Nova Scotia               | ca-ns       |
+| New Brunswick             | ca-nb       |
+| Manitoba                  | ca-mb       |
+| British Columbia          | ca-bc       |
+| Prince Edward Islands     | ca-pe       |
+| Saskatchewan              | ca-sk       |
+| Alberta                   | ca-ab       |
+| Newfoundland and Labrador | ca-nl       |
```
Comment (marcbelmont) [inline source/index.html.md]: The list in complete. For example, ca-yt from the regexp list is missing.

File: source/index.html.md (hunk 1 above)
Comment (yujinyuz) [reply source/index.html.md]: already updated it after realizing. it should be ok now

File: source/index.html.md [hunk 3]
```diff
@@ -631,6 +631,37 @@ Replace `platerecognizer/alpr` by `platerecognizer/alpr:thailand` in the [instal
 | Tocantins           | br-to       |
 
 
+### Canada
+
+| State                     | Region Code |
+| ------------------------- | ----------- |
+| Ontario                   | ca-on       |
```
Comment (marcbelmont) [inline source/index.html.md]: Let's sort that list.

Comment (yujinyuz) [reply source/index.html.md]: done

**Review by marcbelmont — APPROVED** [review]

**Pull request merged**